source/kubernetes/auto-healing.rst

```diff
@@ -17,6 +17,86 @@ self-healing that Kubernetes performs for pods.
 Auto-healing is enabled by default. If desirable, auto-healing can be disabled
 at cluster creation time via the label ``auto_healing_enabled=false``.
 
+****************************
+Support more node conditions
+****************************
+
+There are several built-in node conditions supported by Kubernetes except the
+Ready condition enabled by Catalyst Cloud Kubernetes service auto healing,
+shown as below:
+
++--------------------+----------------------------------------------------------+
+| Node Condition     | Description                                              | 
++====================+==========================================================+
+| Ready              | True if the node is healthy and ready to accept pods,    | 
+|                    | False if the node is not healthy and is not accepting    |
+|                    | pods, and Unknown if the node controller has not heard   |
+|                    | from the node in the last node-monitor-grace-period      |
+|                    | (default is 40 seconds)                                  |
++--------------------+----------------------------------------------------------+
+| DiskPressure       | True if pressure exists on the disk size--that is, if    |
+|                    | the disk capacity is low; otherwise False                |
++--------------------+----------------------------------------------------------+
+| MemoryPressure     | True if pressure exists on the node memory--that is, if  |
+|                    | the node memory is low; otherwise False                  |
++--------------------+----------------------------------------------------------+
+| DiskPressure       | True if pressure exists on the disk size--that is, if    |
+|                    | the disk capacity is low; otherwise False                |
++--------------------+----------------------------------------------------------+
+| PIDPressure        | True if pressure exists on the disk process--that is, if |
+|                    | the disk process is low; otherwise False                 |
++--------------------+----------------------------------------------------------+
+| NetworkUnavailable | True if the network for the node is not correctly        |
+|                    | configured, otherwise False                              |
++--------------------+----------------------------------------------------------+
+| KernelDeadlock     | True if the kernel has no deadlock otherwise False       |
++--------------------+----------------------------------------------------------+
+		
+To support more conditions monitoring and healing, you can just edit the
+configmap named ``magnum-auto-healer-config`` shown as below to support additional
+node conditions.   							
+
+.. code-block:: yaml
+
```

Comment:
If it isn't too much more code, it would be nice to include a full resource definition here (including the `apiVersion`, `kind`, and so on), since people will generally prefer to apply a full resource rather than editing the existing one.

Reply:
(Or, if that's not possible or a bad idea, we could explain why?)

Reply:
Sure, I can do that.